readalongs/cli.py

```diff
@@ -66,116 +73,132 @@ def cli():
-@click.option('-c', '--closed-captioning', is_flag=True, help='Export sentences to WebVTT and SRT files')
-@click.option('-d', '--debug', is_flag=True, help='Add debugging messages to logger')
-@click.option('-f', '--force-overwrite', is_flag=True, help='Force overwrite output files')
-@click.option('-i', '--text-input', is_flag=True, help='Input is plain text (assume paragraphs separated by blank lines, 1 paragraph per page)')
-@click.option('-l', '--language', type=click.Choice(LANGS, case_sensitive=False), help='Set language for plain text input')
-@click.option('-u', '--unit', type=click.Choice(["w", "m"], case_sensitive=False), help='Unit (w = word, m = morpheme) to align to')
-@click.option('-s', '--save-temps', is_flag=True,
-              help='Save intermediate stages of processing and temporary files (dictionary, FSG, tokenization etc)')
-@click.option('-t', '--text-grid', is_flag=True, help='Export to Praat TextGrid & ELAN eaf file')
-@click.option('-x', '--output-xhtml', is_flag=True, help='Output simple XHTML instead of XML')
+@app.cli.command(context_settings=CONTEXT_SETTINGS, short_help="Force align a text and a sound file.")
+@click.argument("inputfile", type=click.Path(exists=True, readable=True))
+@click.argument("audiofile", type=click.Path(exists=True, readable=True))
+@click.argument("output-base", type=click.Path())
+@click.option("-b", "--bare", is_flag=True, help="Bare alignments do not split silences between words")
+@click.option("-c", "--config", type=click.Path(exists=True), help="Use ReadAlong-Studio configuration file")
+@click.option("-C", "--closed-captioning", is_flag=True, help="Export sentences to WebVTT and SRT files")
+@click.option("-d", "--debug", is_flag=True, help="Add debugging messages to logger")
+@click.option("-f", "--force-overwrite", is_flag=True, help="Force overwrite output files")
+@click.option("-i", "--text-input", is_flag=True, help="Input is plain text (assume paragraphs separated by blank lines, 1 paragraph per page)")
+@click.option("-l", "--language", type=click.Choice(LANGS, case_sensitive=False), help="Set language for plain text input")
+@click.option("-u", "--unit", type=click.Choice(["w", "m"], case_sensitive=False), help="Unit (w = word, m = morpheme) to align to")
+@click.option("-s", "--save-temps", is_flag=True, help="Save intermediate stages of processing and temporary files (dictionary, FSG, tokenization etc)")
+@click.option("-t", "--text-grid", is_flag=True, help="Export to Praat TextGrid & ELAN eaf file")
+@click.option("-x", "--output-xhtml", is_flag=True, help="Output simple XHTML instead of XML")
 def align(**kwargs):
-    """Align INPUTFILE and WAVFILE and create output files at OUTPUT_BASE.
+    """Align INPUTFILE and AUDIOFILE and create output files at OUTPUT_BASE.
 
     inputfile : A path to the input text file
 
-    wavfile : A path to the input audio file
+    audiofile : A path to the input audio file
```

Comment:
Here's it's more important to tell the user what audio files are supported, if we can, since this string is part of the help message.